content/en/blog/_posts/2023-04-27-node-expand-secret-moves-beta

```diff
@@ -0,0 +1,168 @@
++---
++layout: blog
++title: "Kubernetes 1.27: CSI node expand secret support moves to Beta"
++date: 2023-04-27
++slug: csi-node-expand-secret-support-beta
++ ---
++
++**Author:** Humble Chirammal, Louis Koo
```

Comment:
```suggestion
---
layout: blog
title: "Kubernetes 1.27: CSI node expand secret support moves to Beta"
date: 2023-04-27
slug: csi-node-expand-secret-support-beta
---

**Authors:** Humble Chirammal, Louis Koo
```

If either author is associated with a company, list those here (see other articles to learn how we usually do it).